docs/http.rst

```diff
@@ -258,30 +261,32 @@ For the following call definitions:
 Allowed bridges are `gcm` (Google Cloud Messaging), `fcm` (Firebase Cloud
 Messaging), and `apns` (Apple Push Notification system)
 
-   :{token}: The bridge specific public exchange token
+   :{client_token}: The bridge specific public exchange token
 
-Each protocol requires a unique token that addresses the remote application.
+Each protocol may require a unique token that addresses the remote application
 For GCM/FCM, this is the `SenderID` and is pre-negotiated outside of the push
-service.
+service. For APNS, there is no client token, so this value is arbitrary.
+Feel free to use "a" or "0" or any other valid URL path token. For our examples, we will use a client token of "33clienttoken33".
 
-   :{instanceid}: The bridge specific private identifier token
+   :{instance_id}: The bridge specific private identifier token
 
 Each protocol requires a unique token that addresses the
 application on a given user's device. This is usually the product of the
```

Comment:
"This is the FCM registration token or APNS device token, generated when the UA registers with the bridge."